components/camera/replaypcd/replaypcd_utils_test.go

```diff
@@ -46,6 +46,7 @@ type mockDataServiceServer struct {
 func (mDServer *mockDataServiceServer) BinaryDataByIDs(ctx context.Context, req *datapb.BinaryDataByIDsRequest,
 ) (*datapb.BinaryDataByIDsResponse, error) {
 	// Parse request
+	//nolint:staticcheck
```

Comment:
this lint shows up on macs only for some reason, I'd ignore it for now.

Reply:
it was showing up on the CI checks so I added this... should I remove it?